src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const rows = table.rows;
+
+for (let i = 0; i < rows.length; i++) {
+  const secondColumn = rows[i].cells[1].textContent;
+  const tableElement = rows[i].insertCell(4);
```

Comment:
The code assumes that each row has at least five cells. If this is not guaranteed, accessing the fifth cell (index 4) might lead to an error. Consider checking the number of cells in each row before inserting a new cell.